dojo/tools/sonarqube_api/importer.py

```diff
@@ -18,7 +19,9 @@ class SonarQubeApiImporter(object):
     """
 
     def get_findings(self, filename, test):
-        return self.import_issues(test)
+        items = self.import_issues(test)
+        items.extend(self.import_hotspots(test))
```

Comment:
Hotspots generate a lot of false positive.
Could you add a feature flag that enable/disable this line?
```suggestion
        if settings.SONARQUBE_API_PARSER_HOTSPOTS:
            items.extend(self.import_hotspots(test))
```

Reply:
You have an example here:
https://github.com/DefectDojo/django-DefectDojo/blob/92a14eac19ab6c53777c802f334d1b9c48a5b54a/dojo/settings/settings.dist.py#L165-L167

Reply:
Done